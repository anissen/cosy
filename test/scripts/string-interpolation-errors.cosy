print 'answer: ' + 3
print 42 + true