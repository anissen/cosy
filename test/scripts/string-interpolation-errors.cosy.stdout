[line 1] Error at "+": Values of type Str and Num cannot be concatinated.
[line 1] Hint: Use string interpolation, e.g. {value}, to add non-string types to a string.
[line 2] Error at "+": Values of types Num and Bool cannot be added.
